tools/PI/DevHome.PI/ViewModels/WinLogsPageViewModel.cs

```diff
@@ -137,25 +133,24 @@ private void StopWinLogs(bool shouldCleanLogs = true)
         }
     }
 
-    private void WinLogsOutput_CollectionChanged(object? sender, NotifyCollectionChangedEventArgs e)
+    private void FindPattern(string message)
```

Comment:
I'm confused.  What pattern is this searching for?  I would think that a "FindPattern" method would accept a pattern to find.